### DISCUSSION TITLE
Is elasticsearch7.16.2 supported?

### DISCUSSION THREAD

**Discussion by chaoqingshuai** [jertel/elastalert2#1223] [Q&A] [answered]
When pulling and installing Elastalert 2, it automatically uninstalls the local Elasticsearch 7.16.2 and installs version 7.10.1. How can I keep version 7.16.2 and avoid installing 7.10.1?

**Comment by nsano-rururu** [comment, marked as answer]
Are you referring to elasticsearch-py?
Since the docker image is published, please use elastalert2 with docker.
